blazor/diagram/connectors/customization.md

```diff
@@ -734,6 +734,100 @@ The following code example illustrates how to set the connection padding value f
+<SfDiagramComponent @ref="Diagram" Width="1000px" Height="500px" Nodes="@nodes" Connectors="@connectors" EnableConnectorSplitting="true">
+
+</SfDiagramComponent>
+@code {
+    //Reference the diagram
+    SfDiagramComponent Diagram;
+    // Initialize diagram's connector collection
+    DiagramObjectCollection<Connector> connectors = new DiagramObjectCollection<Connector>();
+    // Initialize diagram's node collection
+    DiagramObjectCollection<Node> nodes = new DiagramObjectCollection<Node>();
+    protected override void OnInitialized()
+    {
+        nodes = new DiagramObjectCollection<Node>() {
+          new Node() {  OffsetX = 100,
+            OffsetY = 100,
+            Height = 50,
+            Width = 100,
+            ID = "node1",
+            Style = new ShapeStyle(){ Fill = "#6495ED",
+                StrokeColor = "#6495ED",},
+            Shape = new BasicShape() { Type = NodeShapes.Basic, Shape = NodeBasicShapes.Rectangle }
+          },
+          new Node() {  OffsetX = 300,
+            OffsetY = 300,
+            Height = 50,
+            Width = 100,
+            ID = "node2",
+            Style = new ShapeStyle(){ Fill = "#6495ED",
+                StrokeColor = "#6495ED",},
+           Shape = new BasicShape() { Type = NodeShapes.Basic, Shape = NodeBasicShapes.Rectangle }
+          },
+          new Node() {  OffsetX = 300,
+            OffsetY = 100,
+            Height = 50,
+            Width = 100,
+            ID = "node3",
+            Style = new ShapeStyle(){ Fill = "#6495ED",
+                StrokeColor = "#6495ED",},
+           Shape = new BasicShape() { Type = NodeShapes.Basic, Shape = NodeBasicShapes.Rectangle }
+          }
+        };
+        Connector Connector = new Connector()
+            {
+                ID = "connector1",
+                //Source node id of the connector.
+                SourceID = "node1",
+                TargetDecorator = new DecoratorSettings()
+                {
+                    Style = new ShapeStyle()
+                    {
+                        Fill = "#6495ED",
+                        StrokeColor = "#6495ED",
+                    }
+                },
+                //Target node id of the connector.
+                TargetID = "node2",
+                Style = new ShapeStyle()
+                {
+                    Fill = "#6495ED",
+                    StrokeColor = "#6495ED",
+                },
+                // Type of the connector
+                Type = ConnectorSegmentType.Straight,
+                Constraints = ConnectorConstraints.Default | ConnectorConstraints.AllowDrop,
+            };
+        connectors.Add(Connector);
+    }
+}
+```
+
+>**Note:** The 'AllowDrop' constraints must be enabled for the connector to allow dropping a node.
+
+Similarly, when dropping a node onto the target connector, the node connection differs based on the state of the connection and we can create a new connection by dropping on the connector 
```

Comment:
Cannot understand this paragraph. Can you explain what is this purpose?

Reply:
To explain that depending on the state of connection of the connector means it has either source node only or target node only or both nodes the behaviour will differ upon dropping the node on that connector. we have the explanation for that in the next heading for "Create connection using Connector splitting"